core/src/main/java/nu/yona/server/messaging/service/DiscloseResponseMessageDTO.java

```diff
@@ -53,6 +53,12 @@ public String getNickname()
 		return nickname;
 	}
 
+	@Override
+	public boolean canBeDeleted()
```

Comment:
We should implement a test to verify deletion of such messages. In the original code they could not be deleted, but that wasn't noticed.